code/modules/projectiles/guns/ds13/contactbeam.dm

```diff
@@ -252,12 +254,12 @@
 
 /obj/item/weapon/cell/contact
 	name = "contact energy"
-	desc = "A heavy power pack designed for use with the C99 Supercollider Contact Beam"
+	desc = "A heavy power pack designed for use with the C99 Supercollider Contact Beam."
 	origin_tech = list(TECH_POWER = 6)
 	icon = 'icons/obj/ammo.dmi'
 	icon_state = "contact_energy"
 	w_class = ITEM_SIZE_LARGE
-	maxcharge = 5000
+	maxcharge = 4000
```

Comment:
only 4 shots per battery? that seems really restrictive, it'd be better to tweak the damage and accuracy values